week-1/J-functions/exercise.js

```diff
@@ -1,7 +1,12 @@
 function halve(number) {
   // complete the function here
+  return number/2;
 }
 
 var result = halve(12);
 
 console.log(result);
+// calling with different value
+var result = halve(122);
```

Comment:
You have already "declared" the variable called result in line 5. If you want to re-use this, you shouldn't *declare* it again, but just *assign*. So, line 8 should read `result = halve(24);`

Reply:
Yes, thanks for spotting! I did not realise.